### DISCUSSION TITLE
Add a 'no action' bind to stylus binds

### DISCUSSION THREAD

**Discussion by tamewild** [linuxmint/discussions#189] [Ideas]
Hi all! I've been loving Linux Mint, but there's just one incredibly tiny infinitesimal gripe I have -- I can't unbind my tablet's stylus side buttons (through the settings interface) so I have to resort to running commands every time I plug in my device. I know you guys have much more important priorities than my very very insignificant gripe but yeah.. I hope someday I can just unbind it directly in the settings app. ([I think GNOME supports it](https://help.gnome.org/users/gnome-help/stable/wacom-stylus.html.en), but I don't use GNOME)

**Comment by mtwebster** [comment]
Do  you  mean you want to have a 'disabled' option in the combo boxes here?
![image](https://github.com/linuxmint/discussions/assets/262776/8892df32-67f8-4cd9-824f-b26e1c3c8447)

**Reply by tamewild** [reply]
yes!

**Comment by mtwebster** [comment]
implemented:
https://github.com/linuxmint/muffin/commit/2265985ff36744a751d402641f3e77cbfcb49b4d

**Reply by tamewild** [reply]
wow!! that was so fast, thank you so much!! cant wait for it to land in a future update! (im new to all of this, i presume thats how it works)
